man/systemd.exec.xml

```diff
@@ -271,6 +271,25 @@
         <xi:include href="system-only.xml" xpointer="singular"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RootExtensions=</varname></term>
+
+        <listitem><para>Takes a space-separated list of paths to block device nodes or regular files as argument.
+        For each image located in such paths, a variable overlay of <filename>/usr/</filename> and
+        <filename>/opt/</filename> is set up on the root. Nodes in the image that are leaves in the root (empty
```

Comment:
Not exactly related but /opt is not documented at all in the file-hierarchy man page. 
So why treat it as a special case ? what is supposed to go in there ?

Reply:
It is documented in the FSH 3.0: https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s13.html
It's for non-distro-vendor extensions, commonly used for binaries (eg: chrome is installed in /opt). So it's a third-party /usr, so to speak.

Reply:
I thought /usr/local was the place for that, but ok...

Reply:
The /usr/local hierarchy is for programs built locally and installed by the local system administrator/user